phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -2231,6 +2231,7 @@ public void createTable(RpcController controller, CreateTableRequest request,
                                     PhoenixDatabaseMetaData.SYSTEM_CATALOG_NAME_BYTES,
                                     remoteMutations, fullTableName,
                                     MetaDataProtos.MutationCode.UNABLE_TO_UPDATE_PARENT_TABLE);
+                        clearParentTableFromCache(clientTimeStamp, table);
```

Comment:
Why explicitly we need to clear the cache, doesn't the server cache holds the current table always?